### DISCUSSION TITLE
How to setup monorepo, but with docs directory handled as separate project?

### DISCUSSION THREAD

**Discussion by Igloczek** [pnpm/pnpm#5892] [Q&A]
Hi 👋 

I'm trying to set up a monorepo, but with a one directory to act as a separate app.

I'd like to be able to run `cd docs && pnpm install` and actually execute it inside the `docs` directory, not in the project root, with own `pnpm-lock.yaml`, and without installing this stuff when running `pnpm install` from the root.

Structure:
```
├── docs
|   ├──package.json
├── packages
|   ├── package-a
|   ├── package-b
├── package.json
├── pnpm-workspace.yaml
```

In `pnpm-workspace.yaml` I have only `packages/*`.

Is there some flag/config, that I can set to get what I need?

I tried with [recursive-install](https://pnpm.io/npmrc#recursive-install) inside the `docs`, but then it throws some weird message `No projects matched the filters in [my local path]"

**Comment by zkochan** [comment]
Put an empty `pnpm-workspace.yaml` file to the `docs` directory. You'll be able to run `pnpm install` there.
